src/redux/selectors/benefits/benefitsDataSelectors.js

```diff
@@ -24,6 +24,8 @@ export const benefitTransformer = (data, lang) => {
         : `select ${data["title"]}`,
     isSelected: data.isSelected,
     isEligible: data.isEligible,
+    serviceType: data["service_type"],
+    benefitTag: data["service_type"],
```

Comment:
benefitTag is getting the wrong data, shout be "benefit_tag"

Reply:
We're using the tag to distinguish if its an internal or external benefit. This is correct I think

Reply:
If that is the case, what is the purpose of serviceType?

Reply:
It is being used to determine which grid the benefit should appear. External vs Internal... Not the best implementation but find for prototype/demo